raindrops/ena/raindrops.go

```diff
@@ -0,0 +1,20 @@
+package raindrops
+
+import "strconv"
+
+func Convert(number int) string {
+	ret := ""
```

Comment:
I'd use a named return here.

Reply:
IMO it's less readable, even for a short function. The most confusing part is the naked return.

Reply:
... what's even worse with that "feature" is it's inconsistent. While the above code will work if the return value is a simple type, but not if it's a map for example as it declares nil map which can't be changed later in the func body https://go.dev/play/p/gmJPN2meo-J